### ISSUE TITLE
Make sure `intro` is at the top of the module wrapper

### ISSUE THREAD

**Issue by ericf** [rollup/rollup#880]
I received [an issue](https://github.com/yahoo/react-intl/issues/616) where someone encountered a cryptic error message:
```
Uncaught TypeError: Cannot use 'in' operator to search for 'default' in undefined
```

Which was caused by this line in [ReactIntl's Rollup-built dist file](https://npmcdn.com/react-intl@2.1.4/dist/react-intl.js):
```js
var React__default = 'default' in React ? React['default'] : React;
```
_**Note:** React is configured as a `globals` in the Rollup config._

To improve this I figured I'd add an `intro` to type-check `React` and provide a more helpful error message, but the problem is my `intro` was inserted _after_ the line above which is the first line in the UMD factory function.

It would be great if the `intro` was guaranteed to be the first thing inside the module wrapper, even when a default `import` statement is used for one of the build's `globals`. To get around this I was replace the default import statement with `import * as React`: https://github.com/yahoo/react-intl/pull/617#discussion_r76280825 after doing this, my `intro` was the first thing in UMD factory function since the `React__default` line didn't need to be generated.

**Comment by Rich-Harris** [comment]
Fixed in 0.34.11 – thanks

**Comment by juliacochran** [comment]
@ericf can you explain more about how you solved this? I am having the same issue with the new react `prop-types` packaing using rollup `^0.41.4`
